update.bash

```diff
@@ -6,15 +6,20 @@ if [[ "$(id -u)" -ne 0 ]]; then
   echo "ERROR: You need to be ROOT (sudo can be used)."
   exit 1
 fi
-if ! [[ -f /usr/local/sbin/zram-config ]]; then
+if ! [[ -f /usr/local/sbin/zram-config || -f /usr/sbin/zram-config ]]; then
   echo -e "ERROR: zram-config is not installed.\\nPlease run \"sudo ${BASEDIR}/install.bash\" to install zram-config instead."
   exit 1
 fi
 
-if ! dpkg -s 'gcc' 'make' 'libc6-dev' &> /dev/null; then
+if ! dpkg -s 'gcc' 'make' 'libc6-dev' &> /dev/null && ! [[ "$(grep -o '^ID=.*$' /etc/os-release | cut -d'=' -f2)" == "alpine" ]]; then
   echo "Installing needed packages (gcc, make, libc6-dev)"
   apt-get install --yes gcc make libc6-dev || exit 1
 fi
+if ! [[ "$(apk info 2> /dev/null | grep -E '^(gcc|make|fts-dev|linux-headers|util-linux-misc|musl-dev)' | tr '\n' ' ')" == "fts-dev gcc make util-linux-misc musl-dev linux-headers " ]]; then
+  echo "Installing needed packages (gcc, make, fts-dev, linux-headers, util-linux-misc, musl-dev)"
+  apk add gcc make fts-dev linux-headers util-linux-misc musl-dev || exit 1
+fi
+
```

Comment:
The `coretutils` package needs to also be included here I think. I discovered that Alpines default BusyBox `mv` command does not have `-T` flag which is used by overlay-tools. The errors were sometimes not very obvious in the zram-config logs, but during overlay merges when using DIR mode, lower files will get deleted, but upper files won't take their place...! Very bad, anything you edit while using zram will get zapped when you shut down the service :(

Not sure if there is another way to update only the `mv` command for BusyBox?

For reference these are the other files included in `coreutils`:
https://pkgs.alpinelinux.org/contents?file=&path=&name=coreutils&branch=edge&repo=main&arch=x86_64

Reply:
Even then does the mv command point to the correct version?

Reply:
Hum thats starnge! My Alpine (and busybox) support the `-T` flag!
```
BusyBox v1.34.1 (2022-02-02 18:21:20 UTC) multi-call binary.

Usage: mv [-finT] SOURCE DEST
or: mv [-fin] SOURCE... { -t DIRECTORY | DIRECTORY }

Rename SOURCE to DEST, or move SOURCEs to DIRECTORY

	-f	Don't prompt before overwriting
	-i	Interactive, prompt before overwrite
	-n	Don't overwrite an existing file
	-T	Refuse to move if DEST is a directory
	-t DIR	Move all SOURCEs into DIR
```